_posts/2025-05-27-rebar3_plugin.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Emergency bug fix release `atomvm_rebabr3_plugin 0.7.5` has been published
+author: The AtomVM Team
+excerpt_separator: <!--more-->
+---
+
+## 2025/05/27 Emergency bug fix release `atomvm_rebabr3_plugin 0.7.5` is available
+
+We have released [atomvm_rebabr3_plugin 0.7.5](https://github.com/atomvm/atomvm_rebar3_plugin/releases/tag/0.7.5)
+as an emergency bug fix for the broken dependencies when pulling
+[atomvm_rebar3_plugin from hex.pm](https://hex.pm/packages/atomvm_rebar3_plugin).
+
+This was due to the uf2tool dependency source from the GitHub tag rather than the hex package.
+Appologies to anyone who tried to use the `atomvm_rebabr3_plugin 0.7.4` release from hex.pm.
```

Comment:
Apologies

Reply:
Thanks! I obviously forgot to run a spell check.